roles/repository/README.md

```diff
@@ -0,0 +1,94 @@
+# redhat_cop.ah_configuration.repository
+## Description
+An Ansible Role to create Namespaces in Automation Hub.
```

Comment:
```suggestion
An Ansible Role to create Repositories in Automation Hub.
```

I realise thats wrong in the role you copied from too. Feel free to raise a PR for that too!